### DISCUSSION TITLE
RDKit bond type - query

### DISCUSSION THREAD

**Discussion by Suganesh-repo** [rdkit/rdkit#7167] [Q&A] [answered]
BondType: Double 
Bond Direction:Either double

works well if i replace a bond, but if i attach a bond at some atom point, it comes only as double bond..

<img width="183" alt="Screenshot 2024-02-16 at 1 30 00 AM" src="https://github.com/rdkit/rdkit/assets/99866614/548191a9-ffa0-4794-ae07-d1f36c69ffd0">
<img width="245" alt="Screenshot 2024-02-16 at 1 30 04 AM" src="https://github.com/rdkit/rdkit/assets/99866614/623d3b1f-c8b1-49d2-83d9-183d2c9e4334">

**Comment by ptosco** [comment, marked as answer]
1. The second molecule is chemically incorrect as carbon 0 is pentavalent - you cannot have an _exo_ double bond on an aromatic ring.
2. The bond does not show up as `EITHERDOUBLE` because it would not make sense: your system is symmetric, and therefore there is no possibility for geometric isomerism. If you introduce an asymmetry, the bond will show up as `EITHERDOUBLE`. However, the carbon is still pentavalent and hence that molecule is chemically incorrect.
```
m = Chem.MolFromSmiles("CC=C1=CC=CC=C1", sanitize=False)
m.GetBondWithIdx(1).SetBondDir(Chem.BondDir.EITHERDOUBLE)
m
```
![image](https://github.com/rdkit/rdkit/assets/5244385/856d1cc8-11ca-494e-8d0f-dd7a61cfced1)

**Reply by Suganesh-repo** [reply]
Thanks for the reply! 
Pls do check this too.
 https://github.com/rdkit/rdkit/discussions/7164
 @ptosco
